frontend/src/cljs/imas_seamap/map/events.cljs

```diff
@@ -207,7 +220,8 @@
       ;; If category is contours, turn it into bathymetry but with a contours attribute set:
       (as-> lyr (if (= (:category lyr) :contours) (assoc lyr :contours true) lyr))
       (update :category    #(if (= % :contours) :bathymetry %))
-      (update :server_type (comp keyword string/lower-case))))
+      (update :server_type (comp keyword string/lower-case))
+      (update :server_url #(if (= % "https://geoserver-dev.imas.utas.edu.au/geoserver/wms") "https://geoserver.imas.utas.edu.au/geoserver/wms" %))))
```

Comment:
What's going on here; is this debug code that snuck through?

Reply:
Whoops, yes!

Reply:
Well, that's been removed immediately.